src/ui/components/ListaProductos/filter/FormControlCategory.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { FormControl,FormLabel,FormGroup,FormControlLabel,Typography,Divider} from '@mui/material'
+import BpCheckbox from './BpCheckedIcon'
+import CustomTypographySub from './CustomTypography'
+import {defaultStyleFormItem ,DefaultTypographyItem,StylesFormGroupCategory} from './stylesFilter'
+
+export const FormControlCategory = () => {
+  return (
+    <FormControl component="fieldset" variant="standard">
+        <FormGroup sx={StylesFormGroupCategory}>
+        <FormLabel><CustomTypographySub subtitle={"Categoria"}/></FormLabel>
+          <FormControlLabel control={<BpCheckbox name="mujer" checked/>} label={
+              <Typography  sx={{...DefaultTypographyItem,color:'secondary.main',width:'38px'}}>
+                Mujer
+              </Typography>} 
+          sx={{...defaultStyleFormItem,width:'66px'}}
```

Comment:
the better way to set up the breakpoint is using our function mq from the css 
https://github.com/DanX3N0/frontend-rojo/blob/main/src/config/mq.js